scripts/exwafs_grib2_0p25_blending.sh

```diff
@@ -2,152 +2,100 @@
-#              - dbn_alert subtype is changed from gfs to WAFS
-#              - Fix bugzilla 1213: Filename should use fHHH instead of FHH.
-#              - Parallel run for each forecast hour, changed from sequential run.
+#  History:  10/02/2024 - WAFS separation
+#              - MPMD parallel run for each forecast hour, changed from sequential run.
 #              - Fix bugzilla 1593: Improve email notification for missing UK WAFS data.
-#              - Extend waiting time window from 15 to 25 minutes
-#              - usonly.emailbody is differentiated for each forecast hour with missing UK data.
-#              - Remove the condition of sending UK unblended data if US unblended data is missing. It won't happen because the job itself won't get triggered if US unblended data is missing.
+#              - Fix bugzilla 1226: Eliminate the duplicated dbn_alert for unblended wafs data
 ################################################################################
 
 set -x
 
 cd "${DATA}" || err_exit "FATAL ERROR: Could not 'cd ${DATA}'; ABORT!"
 
-SEND_UNBLENDED_US_WAFS="NO"
-
-###############################################
-# Specify Timeout Behavior for WAFS blending
-###############################################
-# SLEEP_TIME - Amount of time (secs) to wait for a input file before exiting
-# SLEEP_INT  - Amount of time (secs) to wait between checking for input files
-SLEEP_TIME=${SLEEP_TIME:-1500}
-SLEEP_INT=${SLEEP_INT:-10}
-SLEEP_LOOP_MAX=$((SLEEP_TIME / SLEEP_INT))
-
-##########################
-# look for UK WAFS data.
-##########################
-for ((ic = 1; ic <= SLEEP_LOOP_MAX; ic++)); do
-    # Three(3) unblended UK files for each cycle+fhour: icing, turb, cb
-    ukfiles=$(find "${COMINuk}" -name "egrr_wafshzds_unblended_*_0p25_${PDY:0:4}-${PDY:4:2}-${PDY:6:2}T${cyc}:00Z_t${fhr}.grib2" | wc -l)
-    if ((ukfiles >= 3)); then
-        echo "Found all 3 UK WAFS GRIB2 files, continue ..."
-        break
-    fi
+fhours=($(seq -s ' ' -f "%03g" 6 1 24; seq -s ' ' -f "%03g" 27 3 48))
+np=${#fhours[@]}
+MPIRUN="mpiexec -np ${np} -cpu-bind verbose,core cfp"
 
-    if ((ic == SLEEP_LOOP_MAX)); then
-        products="cb ice turb"
-        for prod in ${products}; do
-            ukfile="egrr_wafshzds_unblended_${prod}_0p25_${PDY:0:4}-${PDY:4:2}-${PDY:6:2}T${cyc}:00Z_t${fhr}.grib2"
-            if [[ ! -f "${COMINuk}/${ukfile}" ]]; then
-                echo "WARNING: UK WAFS GRIB2 file '${ukfile}' not found after waiting over ${SLEEP_TIME} seconds"
-                echo "${COMINuk}/${ukfile}" >>missing_uk_files
-            fi
-        done
-        echo "WARNING: UK WAFS GRIB2 unblended data is not completely available, no blending"
-        SEND_UNBLENDED_US_WAFS="YES"
-        break
-    else
-        sleep "${SLEEP_INT}"
-    fi
+rm -f wafsgrib2_0p25.cmdfile
+for fhr in ${fhours[@]}; do
+    echo "${USHwafs}/wafs_grib2_0p25_blending.sh $fhr > $DATA/${fhr}.log 2>&1">> wafsgrib2_0p25_blending.cmdfile
+    export MP_PGMMODEL=mpmd
```

Comment:
exporting MP_PGMMODEL is independent of the forecast hour.  All it is doing inside the loop, is setting itself over and over.
```suggestion
export MP_PGMMODEL=mpmd
for fhr in ${fhours[@]}; do
    echo "${USHwafs}/wafs_grib2_0p25_blending.sh $fhr > $DATA/${fhr}.log 2>&1">> wafsgrib2_0p25_blending.cmdfile
```